readthedocs/api/v3/views.py

```diff
@@ -680,7 +681,8 @@ class OrganizationsProjectsViewSet(
     NestedViewSetMixin,
     OrganizationQuerySetMixin,
     FlexFieldsMixin,
-    ReadOnlyModelViewSet,
+    ListModelMixin,
+    GenericViewSet,
```

Comment:
This was allowing `/organizations/projects/{project}` which we don't want, since all project endpoints are under `/projects/{project}`.